OsmAnd/src/net/osmand/plus/onlinerouting/OnlineRoutingHelper.java

```diff
@@ -272,4 +282,13 @@ private void saveCacheToSettings() {
 			settings.ONLINE_ROUTING_ENGINES.set(null);
 		}
 	}
+
+	public boolean wasRescueTrackEngineUsed() {
```

Comment:
I don't think we wanted specific for rescuetrack, that's not correct. We probabloy need this functionality for all online routers that enabled GPX approximation

Reply:
done